sql/builder/builder.go

```diff
@@ -66,7 +68,8 @@ func FilterFrom(operations Operations) string {
 		} else {
 			queryBuilder.WriteString(" and")
 		}
-		queryBuilder.WriteString(" " + string(op.Field) + " " + string(op.Token) + " ?" + strconv.Itoa(i+1))
+		queryBuilder.WriteString(" " + string(op.Prefix) + string(op.Field) + " " + string(op.Token) +
+			" ?" + strconv.Itoa(i+1))
```

Comment:
Maybe
```suggestion
		fmt.Fprintf(&queryBuilder, " %s%s %s ?%d", op.Prefix, op.Field, op.Token, i+1)
```

Reply:
Also `queryBuilder.WriteString(fmt.Sprintf(" %s %v", string(m.Key), m.Value))` can be simplified likewise using `fmt.Fprintf`